main/sw/inc/shellio.hxx

```diff
@@ -367,32 +363,32 @@ public:
     void   SetBaseURL( const String& rURL );
 
 	sal_Bool   IsOld() const;
-	sal_uLong  ConvertToNew();				// Textbausteine konvertieren
+	sal_uLong  ConvertToNew();				// convert AutoText entries (ger: Textbausteine)
```

Comment:
Could Textbaustein be translated as "text building block"?
I have no idea what it would mean... in both languages, though

Reply:
Well, yea Leo translates Textbaustein to Text building blog or boilerplate, but in the OpenOffice help it is translated to AutoText entry, which is a feature of OpenOffice.

Reply:
Fun fact: `SwTextBlocks::ConvertToNew()` is _commented out_.
Source: main/sw/source/core/swg/swblocks.cxx

Reply:
Ok, you are right. The correct translation is autotext. Sorry for the noise.

Reply:
No reason to be sorry. It is assuring that you agree.